### ISSUE TITLE
Vuetify template is using large amounts of memory in JupyterLab

### ISSUE THREAD

**Issue by aagrosch** [mariobuikhuizen/ipyvuetify#62]
I have been having issues with `ipyvuetity.VuetifyTemplate` in JupyterLab. Rendering a custom template eats up enormous amounts of CPU usage and memory. Below is the stack trace of me trying to render the menu template example from the example notebook 5 times.

<img width="1552" alt="Stack trace of memory issues with VuetifyTemplate in JupyterLab" src="https://user-images.githubusercontent.com/38324489/79775612-ce3f1480-8334-11ea-9819-ad12f05c0d1f.png">

Comparing the stack trace from an earlier setup with identical code, which does not have the issue.
<img width="1552" alt="Stack trace of working VuetifyTemplate in JupyterLab" src="https://user-images.githubusercontent.com/38324489/79775737-f890d200-8334-11ea-82d4-d4c0f27787a6.png">

With more complicated custom templates, this can cause my browser to use up several GBs of RAM, which is less than ideal. This issue only occurs in JupyterLab however, not in the classic notebook.

If I run `npm list` in the `share/jupyter/lab/staging/node_modules/jupyter-vuetify` directory, the output from the working version looks like this:
```
jupyter-vuetify@1.1.1
├── UNMET DEPENDENCY @jupyter-widgets/base@^1.0.0 || ^2.0.0
├── UNMET DEPENDENCY @mariobuikhuizen/vuetify@2.1.7-alpha.0
├── UNMET DEPENDENCY core-js@^3.0.1
├─┬ jupyter-vue@1.0.2
│ ├── UNMET DEPENDENCY @jupyter-widgets/base@^1.0.0 || ^2.0.0
│ ├── UNMET DEPENDENCY @mariobuikhuizen/vue-compiler-addon@2.6.10
│ ├── UNMET DEPENDENCY core-js@^3.0.1
│ ├── UNMET DEPENDENCY lodash@^4.17.11
│ └── UNMET DEPENDENCY vue@^2.6.10
├── UNMET DEPENDENCY lodash@^4.17.11
├── UNMET DEPENDENCY material-design-icons-iconfont@^5.0.1
└── UNMET DEPENDENCY typeface-roboto@0.0.54
```
In the version with the memory issue, the output of `npm list` is
```
jupyter-vuetify@1.1.1
├── UNMET DEPENDENCY @jupyter-widgets/base@^1.0.0 || ^2.0.0
├── UNMET DEPENDENCY @mariobuikhuizen/vuetify@2.1.7-alpha.0
├── UNMET DEPENDENCY core-js@^3.0.1
├─┬ jupyter-vue@1.3.1
│ ├─┬ @jupyter-widgets/base@3.0.0
│ │ ├── UNMET DEPENDENCY @jupyterlab/services@^5.0.0
│ │ ├── UNMET DEPENDENCY @lumino/coreutils@^1.2.0
│ │ ├── UNMET DEPENDENCY @lumino/messaging@^1.2.1
│ │ ├── UNMET DEPENDENCY @lumino/widgets@^1.3.0
│ │ ├── UNMET DEPENDENCY @types/backbone@^1.4.1
│ │ ├── UNMET DEPENDENCY @types/lodash@^4.14.134
│ │ ├── UNMET DEPENDENCY backbone@1.2.3
│ │ ├── UNMET DEPENDENCY base64-js@^1.2.1
│ │ ├── UNMET DEPENDENCY jquery@^3.1.1
│ │ └── UNMET DEPENDENCY lodash@^4.17.4
│ ├─┬ @mariobuikhuizen/vue-compiler-addon@2.6.10-alpha.1
│ │ └── UNMET DEPENDENCY de-indent@^1.0.2
│ ├── UNMET DEPENDENCY core-js@^3.0.1
│ ├── UNMET DEPENDENCY lodash@^4.17.11
│ ├── UNMET DEPENDENCY uuid@^3.4.0
│ └── UNMET DEPENDENCY vue@^2.6.10
├── UNMET DEPENDENCY lodash@^4.17.11
├── UNMET DEPENDENCY material-design-icons-iconfon
```
Perhaps interesting is, that in the newer version of `jupyter-vue` (1.3.1 vs 1.0.2), an alpha version of the `vue-compiler-addon` (`@mariobuikhuizen/vue-compiler-addon@2.6.10-alpha.1`) gets installed.

These have been the only differences we have been able to spot between the two setups so far, after making sure that we have kept everything else identical.

**Comment by antmanler** [comment]
Occurred the same issue, I think may caused by switching to lodash https://github.com/mariobuikhuizen/ipyvue/commit/69a6728849c05b71d81cd52822cdd9bdc78dcbc5 which introduced more CPU cycles and potential Mem leaks.

**Comment by mariobuikhuizen** [comment]
Thanks for the detailed analysis @aagrosch. I can reproduce the issue and will try to fix it.

**Comment by mariobuikhuizen** [comment]
I've released ipyvue v1.3.2 which fixes this issue for me. Can you try if it does for you too?

**Comment by aagrosch** [comment]
Great, that has fixed the issue for me, so I'm closing it. Thanks!
